examples/break-statement/break_statement.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/io;
+
+public function main() {
+
+    string[] names = ["Bob", "Jo", "Ann", "Tom"];
+    int i = 0;
+    while true {
+        io:println(names[i]);
+        // loop breaks when condition satisfied
+        if names[i] == "Tom" {
+            break;
+        }
+
+        i += 1;
+    }
+
+    foreach string name in names {
+        // loop breaks when condition satisfied
+        if name == "Ann" {
+            break;
+        }
+
+        io:println(name);
+    }
+}
```

Comment:
I think since we are only focusing on `break;`, it is better to have a simple example like this.
```ballerina
public function main() {

    foreach int i in 0...9 {
        if (i > 5) {
            break;
        }
        io:println(i);
    }

    int i = 0;
    while true {
        if i > 5 {
            break;
        }
        io:println(i);
        i = i + 1;
    }
}
```
Let's discuss this further.

Reply:
This example is an extension of previous examples used in `while` and `foreach` BBEs. But if we are considering the simplicity we can replace the existing example with the suggested implementation, @hasithaa WDYT ?

Reply:
+1, let's do what Kavindu suggested. Can you send a sperate PR? I want to merge the current one for the dev preview.